remoteworkitem/remoteworkitem.go

```diff
@@ -182,3 +184,12 @@ func Map(item AttributeAccessor, mapping WorkItemMap) (app.WorkItem, error) {
 	}
 	return workItem, nil
 }
+
+func getConfigurationData() *configurationHandler.ConfigurationData {
```

Comment:
This could be a helper constructor in the configuration package instead. And the configuration options should be passed in from main, not created inside here.